### PR TITLE
Fix : Embed training videos #14 and #15 in Part 1 Set 2 page of the Gitbook learning resource

### PR DESCRIPTION
**Pull request by oyenuga17**
**This pull request makes the following changes:**
* Fixes issue #46  

**Checklist:**
- [x] File or folder now contains changes as specified in the issue i worked on
- [x] I have linked the issue I worked on to this pull request submitted by me
- [x] I certify that I ran my checklist.

Ping @Ifycode/git-github-training-resource

### CODE REVIEW

**Comment by oyenuga17** [comment]
Task done please preview. @Ifycode

**Comment by Ifycode** [comment]
Senior woman! 🙌🏽

File: doc/part-1-github-ui/set-2-github-ui-learning-videos-and-tasks.md [hunk 1]
```diff
@@ -5,8 +5,17 @@ PART 1: GITHUB UI
...
-Set 2 learning videos are not available yet. Check back on or before the 12th of October 2022.
+{% embed url="https://www.youtube.com/watch?v=hRFBG3lMq9k" %}
+YouTube Video
+{% endembed %}
+
+### Workflow common practices and best practices
+
+{% embed url="https://www.youtube.com/watch?v=Ej396Vra1oQ" %}
+YouTube Video
+{% endembed %}
```
Comment (Ifycode) [inline doc/part-1-github-ui/set-2-github-ui-learning-videos-and-tasks.md:18]: Neatly done! @oyenuga17 
One more thing before I approve, let the pull request title start with `Fix: `

**Comment by oyenuga17** [comment]
😂😂ahh.. I’m learning o ..

**Review by Ifycode — CHANGES_REQUESTED** [review]

File: doc/part-1-github-ui/set-2-github-ui-learning-videos-and-tasks.md (hunk 1 above)
Comment (oyenuga17) [reply doc/part-1-github-ui/set-2-github-ui-learning-videos-and-tasks.md:18]: Alright

Comment (oyenuga17) [reply doc/part-1-github-ui/set-2-github-ui-learning-videos-and-tasks.md:18]: I just did it..

**Review by Ifycode — APPROVED** [review]

**Pull request closed without merging**